strings.txt

```diff
@@ -9261,6 +9261,40 @@ SETUP_RELEASE_TYPES_DESC
 	FR	Vous pouvez spécifier quels types de sorties (voir <a href="https://musicbrainz.org/doc/Release_Group/Type">MusicBrainz</a>) doivent être inclus dans la liste des albums. Si aucun type n'a été défini pour une sortie, le type "Album" est pris par défaut. Le type "Album" ne peut pas être exclu.
 	NL	Je kunt opgeven welke releasetypes je wilt opnemen in de albumlijst (zie <a href="https://musicbrainz.org/doc/Release_Group/Type">MusicBrainz</a>). Als voor een release geen type is gedefinieerd, wordt uitgegaan van 'Album'. Albums kunnen niet van die lijst worden uitgesloten.
 
+SETUP_RELEASE_TYPES_GROUP_OPTIONS
+	EN	Release type grouping options
+	FR	Options de regroupement des types de sorties
+	DE	Gruppierungsoptionen für Release-Typen
+	NL	Groeperingsopties voor het releasetype
+
+SETUP_RELEASE_TYPES_GROUP_OPTIONS_DESC
+	EN	Create a Composer Albums group - always, never, or for certain genres. If due to this setting, certain composer roles are not included in the Composer Albums group, they will appear in a separate "Compositions" group instead, which will list individual tracks, not albums. The genre list should be comma separated.
+	FR	Créez un groupe d'albums de compositeur - toujours, jamais ou pour certains genres. Si, en raison de ce paramètre, certains rôles de compositeur ne sont pas inclus dans le groupe Albums du compositeur, ils apparaîtront à la place dans un groupe « Compositions » distinct, qui répertoriera les pistes individuelles, pas les albums. La liste des genres doit être séparée par des virgules.
+	DE	Erstellen Sie eine Gruppe „Komponistenalben“ – immer, nie oder für bestimmte Genres. Wenn aufgrund dieser Einstellung bestimmte Komponistenrollen nicht in der Gruppe „Komponistenalben“ enthalten sind, werden sie stattdessen in einer separaten Gruppe „Kompositionen“ angezeigt, in der einzelne Titel und keine Alben aufgeführt werden. Die Genreliste sollte durch Kommas getrennt sein.
```

Comment:
Please use regular double quotes `"` instead of low/high quotes for consistency with the remaining German content.

Replace "Genreliste" with "Liste der Stilrichtungen" (again for the sake of consistency).